PowerUp/app/src/main/java/powerup/systers/com/StartActivity.java

```diff
@@ -113,7 +114,7 @@ public void onClick(DialogInterface dialog, int id) {
         aboutButton.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View view) {
-                startActivity(new Intent(StartActivity.this, AboutActivity.class));
+                startActivity(new Intent(StartActivity.this, MemoryMatchTutorialActivity.class));
                 overridePendingTransition(R.animator.fade_in_custom, R.animator.fade_out_custom);
```

Comment:
Seems you're on the test mode here

Reply:
@thisum I will update the PR after #1214 is merged. The duplicate files could be removed only after that.